### PR TITLE
feat(api-client): Download member CSV list

### PR DESCRIPTION
**Pull request by Yserz**

### CODE REVIEW

**Review by bennycode — APPROVED** [review]

File: packages/api-client/src/team/member/MemberAPI.ts [hunk 1]
```diff
@@ -17,12 +17,18 @@
...
+import axios, {AxiosRequestConfig} from 'axios';
 import {ArrayUtil} from '@wireapp/commons';
 
 import {TeamAPI} from '../team/TeamAPI';
-import type {HttpClient} from '../../http/';
+import {handleProgressEvent, HttpClient, ProgressCallback, RequestCancelable, SyntheticErrorLabel} from '../../http/';
 import type {MemberData, Members} from '../member/';
+import {RequestCancellationError} from '../../user';
+
+export interface MemberCSVResponse {
```
Comment (ffflorian) [inline packages/api-client/src/team/member/MemberAPI.ts:28]: Let's move this into its own file

File: packages/api-client/src/team/member/MemberAPI.ts [hunk 2]
```diff
@@ -17,12 +17,18 @@
  *
  */
 
-import type {AxiosRequestConfig} from 'axios';
+import axios, {AxiosRequestConfig} from 'axios';
```
Comment (ffflorian) [inline packages/api-client/src/team/member/MemberAPI.ts:20]:
```suggestion
import {AxiosRequestConfig, CancelToken} from 'axios';
```

File: packages/api-client/src/team/member/MemberAPI.ts [hunk 3]
```diff
@@ -126,4 +133,39 @@ export class MemberAPI {
     const response = await this.client.sendJSON<Members>(config);
     return response.data;
   }
+
+  public async getMemberListCSV(
+    teamId: string,
+    progressCallback?: ProgressCallback,
+  ): Promise<RequestCancelable<MemberCSVResponse>> {
+    const cancelSource = axios.CancelToken.source();
```
Comment (ffflorian) [inline packages/api-client/src/team/member/MemberAPI.ts:141]:
```suggestion
    const cancelSource = CancelToken.source();
```

File: packages/api-client/src/team/member/MemberAPI.ts [hunk 4]
```diff
@@ -126,4 +133,39 @@ export class MemberAPI {
...
+    const config: AxiosRequestConfig = {
+      cancelToken: cancelSource.token,
+      method: 'get',
+      onDownloadProgress: handleProgressEvent(progressCallback),
+      params: {},
```
Comment (ffflorian) [inline packages/api-client/src/team/member/MemberAPI.ts:146]:
```suggestion
      onDownloadProgress: handleProgressEvent(progressCallback),
```

**Review by ffflorian — CHANGES_REQUESTED** [review]

File: packages/api-client/src/team/member/MemberAPI.ts (hunk 3 above)
Comment (Yserz) [reply packages/api-client/src/team/member/MemberAPI.ts]: `'CancelToken' only refers to a type, but is being used as a value here.ts(2693)`

**Review by ffflorian — APPROVED** [review]

**Pull request merged**